config/locales/en.yml

```diff
@@ -377,7 +377,7 @@ en:
       follow: "Follow"
       none: "None"
       users_following_tag: "Users following this tag"
-      ask_question: "Ask a question about <b>%{tag}</b>"
+      subscribe_to_answer: "Ask a question about <b>%{tag}</b>"
       or_subscribe_to_answer: "<a style='margin-bottom:6px;' class='btn btn-default
```

Comment:
@z119 , this line has to be changed. ie 'or_subscribe_to_answer' to 'subscribe_to_answer'